server/api/documents.js

```diff
@@ -20,7 +20,7 @@ router.post('documents.list', auth(), pagination(), async ctx => {
 
   const user = ctx.state.user;
   let where = { teamId: user.teamId };
-  if (collection) where = { ...where, atlasId: collection };
+  if (collection) where = { ...where, collectionId: collection };
```

Comment:
Now we just need to change the api parameters to match :P

Reply:
I actually designed the API to omit `Id` postfix, that's why it wasn't there :)